next.html

```diff
@@ -0,0 +1,625 @@
+                <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+                    <div class="container responsivegrid blurb-content aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div class="cmp-container">
+                            <div class="image img-wrap">
+                                <div class="cmp-image">
+                                    <img src="./assets/blurb-logo.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="lady smiling">
+                                </div>
+                            </div>
+                            <div class="title">
+                                <div class="cmp-title">
+                                        <h1 class="cmp-title__text">Have an Impact as a Young Professional</h1>
+                                </div>
+                            </div>
+                            <div class="text base">
+                                <div data-cmp-is="adaptiveFormText" id="text-53cc58dc93" name="text_demo" data-cmp-adaptiveformcontainer-path="/content/core-components-examples/library/adaptive-form/text/jcr:content/root/responsivegrid/demo/component/guideContainer" class="cmp-adaptiveform-text">
+                                    <div tabindex="0">
+                                        <p>Leaders of impact don't wait until they're older or deeper in their career to make a difference- they start today!</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="container responsivegrid button-wrap aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--12 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div id="blurb-button-wrap" class="cmp-container" style="display: flex;">
+                            <div class="button aem-GridColumn aem-GridColumn--default--6">
+                                <a class="cmp-button" type="button">
+                                    <span class="cmp-button__text">
+                                        Join the Next Mailing List
+                                    </span>
+                                </a>
+                            </div>
+                            <div class="button video-btn">
+                              <a id="button-1d72cecfeb" class="cmp-button" href="/content/leaderimpact/lang/en2/resources/leading-through-a-crisis/video-series.html">
+                                  <span class="cmp-buttonicon cmp-buttonicon--video-btn" aria-hidden="true"></span>
```

Comment:
@Prystal254 These classes should be `cmp-button__icon cmp-button__icon--video-btn`.